### ISSUE TITLE
How to support multiple environments

### ISSUE THREAD

**Issue by justinwhall** [danielcondemarin/serverless-next.js#177]
Hello, is there a way to support multiple environments with the new serverless component? We've upgraded and used to pass env vars to serverless.yml which no longer seems to be an option.

**Comment by danielcondemarin** [comment]
@justinwhall Would this https://github.com/serverless/components#environment-variables help?

**Comment by justinwhall** [comment]
I'm not sure. I'm familiar with passing env vars to the serverless.yml but I'm not certain where they would go with the update?

 Components are a new concept to me. Perhaps I'm missing something here but as far as the `serverless.yml` config goes, I'm not certain how both generate or conditionally deploy to different components – or perhaps that wrong terminology. Same component but different Apps.

What I'm after here is a way to build/deploy different environments something like `sls deploy --env=stage`

Does that make sense?

**Comment by justinwhall** [comment]
I suppose [this](https://github.com/serverless/components/issues/481#issuecomment-527848024) is what I'm looking for. Does this package support something like:

```
myApp:
  component: serverless-next.js
  inputs:
    code: ../
```
With a directory structure like:
```
code
code/prod
code/test
```

**Comment by skycremaschinin** [comment]
Hi,

how could i manage different stages, specifying different cloudfront distributions, buckets and lamba for each stages?

**Comment by danielcondemarin** [comment]
Hi folks, I've just released `@1.3.0` which adds support for custom app directory paths (see [here](https://github.com/danielcondemarin/serverless-next.js#inputs) `nextConfigDir`). Credits to @jayarnielsen for implementing 🙌 

I'm closing the issue as you should now be able to do something like:

```
myApp
│   next.config.js    
│
└───pages
│   │   index.js
│   │   users.js
│   │   ...
│   
└───dev
    │   serverless.yml
└───prod
    │   serverless.yml
``` 

```yml
# dev/serverless.yml
myNextApp:
 inputs:
  domain: [ "dev", "myapp.com" ]
  nextConfigDir: "../"
```
